### PR TITLE
Update ElasticPress Debug bar to 2.1.0

### PR DESCRIPTION
**Pull request by pauarge**
## Description

This PR updates the ElasticPress Debug bar to its latest version, `v2.1.0`.

### Plugin Updated: ElasticPress Debug Bar to 2.1.0

We upgraded ElasticPress Debug Bar to 2.1.0.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. Check out PR.
2. Make sure Enterprise Search is enabled.
3. Perform a search and check its details through the Debug Bar.

### CODE REVIEW

File: debug-bar.php [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 // If the user is an Automattician (typically a vip_support user), then force-enable Debug Bar.
 add_filter( 'debug_bar_enable', function( $enable ) {
+	return true;
```
Comment (rinatkhaziev) [inline debug-bar.php:14]: I think this is a leftover from testing?

Comment (rinatkhaziev) [reply debug-bar.php:14]:
```suggestion
```

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
It seems there's a stray testing call :)

File: debug-bar.php (hunk 1 above)
Comment (pauarge) [reply debug-bar.php:14]: It is a leftover from testing :)

**Review by pschoffer — APPROVED** [review]
Works well.

**Pull request merged**